localization/english/religion/wc_religion_l_english.yml

```
@@ -641,4 +641,16 @@
  cult_of_nightwell_adj:0 "Shal'danalic"
  cult_of_nightwell_adherent:0 "Shal'danalist"
  cult_of_nightwell_adherent_plural:0 "Shal'danalists"
- cult_of_nightwell_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
\ No newline at end of file
+ cult_of_nightwell_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
+
+ trogg_religion:0 "Trogg Religion" #Placeholder
+ trogg_religion_adj:0 "Shal'danalic"
+ trogg_religion_adherent:0 "Shal'danalist"
+ trogg_religion_adherent_plural:0 "Shal'danalists"
+ trogg_religion_desc:0 "Shal'danalism are the Nightwell traditions. Its followers depend on and are changed by the Nighwell's power, the Shal'danalists favor the arcane philosophy of their past as they preserve and evolve their ways in their unending isolation from the outside world."
```

A name idea is here. The description is to be written by some English Proofreader, ping them in the dev.
```suggestion
 trogg_religion:0 "Salamandrism" #Placeholder
 trogg_religion_adj:0 "Salamandric"
 trogg_religion_adherent:0 "Salamandrist"
 trogg_religion_adherent_plural:0 "Salamandrists"
 trogg_religion_desc:0 "We smash and worship salamanders (placeholder)."
```
Source: https://wowpedia.fandom.com/wiki/Trogg#Culture